packages/api/src/external/commonwell/command/cw-patient-data/update-cw-data.ts

```diff
@@ -78,3 +91,9 @@ export async function updateCwPatientDataWithinDBTx(
     { transaction }
   );
 }
+
+const isLinkValid = (link: CwLink, invalidateLinks: CwLink[]): boolean => {
```

Comment:
1. let's not create arrow functions, please - [ref](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#b669ec3473f1414b9d234cbdac20041b)
2. this name and impl is confusing
   - the name doesn't say what its doing
   - it's using inverse booleans a few times, which is hard to follow

Maybe we can rename this to `isContainedAt(CwLink[], CwLink)`, then it becomes

```ts
  const validLinks = invalidateLinks
    ? updatedLinks.filter(link => !isContainedAt(link, invalidateLinks))
    : updatedLinks;
```

It would read even better if we name `invalidateLinks` as `linksToInvalidate`

```ts
  const validLinks = linksToInvalidate
    ? updatedLinks.filter(link => !isContainedAt(link, linksToInvalidate))
    : updatedLinks;
```